src/engraving/rendering/dev/horizontalspacing.cpp

```diff
@@ -385,6 +385,14 @@ void HorizontalSpacing::computeNotePadding(const Note* note, const EngravingItem
         }
     }
 
+    for (Articulation* a : note->chord()->articulations()) {

```

Comment:
This is a bit hacky (and a bit of a performance concern as this is a very hot function). I think the better way to do this would be to add the shape of the LV articulation to its chord. That way, you can simply add to the padding table an Articulation->otherElement entry. At the moment, articulations are never added to segment shapes so LV will be effectively the only articulation item for which that entry is relevant (you can say that in a comment, there are a few other similar cases). In future, I'd like _all_ articulations to be added to the chord shape (at the moment they aren't I think mostly because of skyline-related concerns), because wide articulations should be accounted for in horizontal spacing. When we do that, we'll think about how to specialize the padding for different articulation types

Reply:
Sorted - as articulations are layed out well after horizontal spacing is calculated, this required moving lv layout out of `LayoutArticulations2` and to `layoutPitched` and `layoutTablature`.